compiler+runtime/test/jank/metadata/pass-meta.jank

```diff
@@ -1,11 +1,9 @@
 ; Metadatable
-(assert (= nil (meta {})))
-(assert (= nil (meta [])))
-(assert (= nil (meta '())))
-(assert (= nil (meta 'foo)))
-
-; TODO: Set literals
-;(assert (= nil (meta #{})))
+(assert (map? (meta {})))
+(assert (map? (meta [])))
+(assert (map? (meta '())))
+(assert (map? (meta 'foo)))
+(assert (map? (meta #{})))
```

Comment:
Do the meta maps contain source information?

Reply:
Yep!